src/main/java/pl/agh/edu/iisg/io/vmms/vmmsbackend/model/reservations/Reservation.java

```diff
@@ -0,0 +1,91 @@
+package pl.agh.edu.iisg.io.vmms.vmmsbackend.model.reservations;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.model.User;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.model.VMPool;
+
+import javax.persistence.*;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@Data
+@Entity
+@NoArgsConstructor
+@Table(name = "reservations")
```

Comment:
Nie musimy zmieniać nazw tabeli, w user było to konieczne, ponieważ user jest nazwą zarezerwowaną